### PR TITLE
Implement vale in dev-guide

### LINKED ISSUE #118: Implementing vale tool in dev-guide
Following https://github.com/ansys/ansys-dev/issues/73 I think it's time we start implementing vale in this repository.

Pinging @PipKat for visibility.

The implementation should be something similar to the first commit in https://github.com/ansys/ansys-dev/pull/75 
Many errors and warnings will appear the first time, so we should proceed to review it at the same time we implement it.

### PR DESCRIPTION
**Pull request by RobPasMue**
Closes #118

### CODE REVIEW

**Comment by RobPasMue** [comment]
Implementation done. Required to fix vale errors and warnings before merging. Pinging @PipKat for resolution of issues. You can proceed to solve the vale errors in this same PR/branch, as done in https://github.com.mcas.ms/ansys/ansys-dev/pull/75

You can follow the same procedure: https://github.com/ansys/ansys-dev/pull/75#issuecomment-1158509650

**Comment by RobPasMue** [comment]
Close to 500 errors detected in first run... there is work to be done =)

**Comment by jorgepiloto** [comment]
> Close to 500 errors detected in first run... there is work to be done =)

<div align="center">

**Comment by RobPasMue** [comment]
@PipKat let me know if you can work on this PR as well (similarly to what we did in the ansys/ansys-dev repository). Otherwise, I can try to handle the errors and warnings, no problem. Thanks in advance!

**Comment by PipKat** [comment]
@RobPasMue I can try to resolve them. I was so engrossed in the other one that I hadn't gone through the PR comments that have come through until now. I'll start on this tomorrow, while all I learned is freshed. Thanks!

**Comment by RobPasMue** [comment]
> @RobPasMue Getting all the Vale warnings and errors resolved basically took me all day. The one failing CI check is pointing out an unknown option of "linenos" in a code directive. I had to add "linenos" to Vocab\ANSYS\accept.txt to address a Vale spelling warning. I'm not sure why Vale noted it as a spelling issue given that it is in a code tag, but I've seen this happen elsewhere. I think Vale might have a bug or two (or three)! Lastly, Vale also didn't like the formatting in changelog_file.md. You might want to take a look at what I've done in it.

Hi @PipKat, thanks for the hard work! I managed to solve the problem with the "linenos" code directive. Whenever you have a
`.. code:: {whatever}` command, it is necessary to place a new line in between this line and the next, such that it looks like this:

``` rst
.. code:: yaml

  ...
  ...
```

Instead of like this:
``` rst
.. code:: yaml
  ...
  ...
```

I also modified some broken links which caused the docs build to break (since the title names of some sections changed, so had to change the ref directives, no big deal). I think this PR may be ready for review now. @jorgepiloto could you give it a look whenever possible? I'll try to do the same as well.

File: doc/styles/Vocab/ANSYS/accept.txt
Comment (jorgepiloto) [inline doc/styles/Vocab/ANSYS/accept.txt:74]:
```suggestion
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
Thanks for all the effort implementing this @PipKat and @RobPasMue. The amount of work its impressive!

It looks really great. I just noticed an empty line at the end of the `accept.txt` file. The rest of the files look good to me 👍🏽

**Review by jorgepiloto — APPROVED** [review]
I think this is ready to be merged. Waiting for a confirmation from @RobPasMue and @PipKat in case you want to add something else.

**Comment by RobPasMue** [comment]
LGTM! Merging :)

**Pull request merged**
